packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -30,12 +29,6 @@ const isSheetViewActive = () => !!IModelApp.viewManager.selectedView?.view?.isSh
 export interface MeasureToolsUiProviderOptions {
   itemPriority?: number;
   groupPriority?: number;
-  widgetPlacement?: {
```

Comment:
i dont think youre supposed to remove the entire widget placement config, just the ui1.0 stuff, which should just be the zoneLocation

Reply:
added back